odb_extractor_atoms.py

```diff
@@ -288,7 +721,7 @@ def printseq(sequence, comment='', csv=False, path=''):
         if 'GMOS' in inst:
             coadds = '1'
```

Comment:
This is more a question of personal curiosity: I often see coadds represented as a string. Is there a reason for this? Aren't they always an integer?